pom.xml

```diff
@@ -198,11 +198,18 @@
 			<version>5.1.0.Final</version>
 		</dependency>
 
+		<!-- Testing: Random Pojo Data Generator -->
```

Comment:
Hi @abatac  notice that you have included hibernate-validator in Middleware. 
Please remove this artifact from the projects that use Middleware to avoid possible versions conflicts.